src/cinder/FileWatcher.cpp

```diff
@@ -45,7 +45,7 @@ class Watch : public std::enable_shared_from_this<Watch>, private Noncopyable {
 	void checkCurrent();
 	//! Remove any watches for \a filePath. If it is the last file associated with this Watch, discard
 	void unwatch( const fs::path &filePath );
-	//! Emit the signal callback. 
+	//! Emit the signal callback.
```

Comment:
I'm all for whitespace cleanup but this other file seems out of scope for the PR